tools/config.py

```diff
@@ -241,6 +241,9 @@ def add_config_files(self, flist):
             cfg = json_file_to_dict(f)
             cfg["__config_path"] = full_path
             # If there's already a configuration for a module with the same name, exit with error
+
+            if "name" not in cfg:
```

Comment:
Change is good, but the comment at line 243 (above) is confusing: it was supposed to apply to the next `if` (line 247 beow).